canonical-web-and-design/ubuntu.com
# Subscription loading failed message

## Done

- Display a message if the subscriptions can't be loaded.

## QA

- Open your network dev tools and load [/advantage?test_backend=true](https://ubuntu-com-10386.demos.haus/advantage?test_backend=true) and log in.
- Find the entry for `/advantage/user-subscriptions?test_backend=true`.
- Right click on the entry and click "block" (don't forget to unblock at the end).
- Reload the page.
- The page should load and you should see a message about the subscriptions not loading.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10384.

## Screenshots

<img width="1075" alt="Screen Shot 2021-09-17 at 10 09 57 am" src="https://user-images.githubusercontent.com/361637/133706957-4192448f-1818-4327-a22a-4400bb15e023.png">
